plugins/inspect/groups_inspector.rb

```diff
@@ -16,7 +16,7 @@
 # you may find current contact information at www.suse.com
 
 class GroupsInspector < Inspector
-  def inspect(system, description, options = {})
+  def inspect(system, description, _filter, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.